src/lib/serviceManager.ts

```diff
@@ -63,6 +63,11 @@ export class ServiceManager {
       notifications: new EventEmitter(),
     };
 
+    const svc = this.manager.getService(serviceName, version, env);
+    if (svc && svc.active && svc.active.state === "running") {
+      return svc.active;
```

Comment:
this should return a promise, maybe we can move it down?

Reply:
it does async automatically returns a promise

Reply:
oh. well then. lol

Reply:
yeah wraps return in promise